icoscp/stilt/stiltstation.py

```diff
@@ -504,11 +559,14 @@ def find(**kwargs):
         Remember, that only year and month is checked.
     Example: station.find(dates=['2020-01-01', '2020/05/23'])
 
+    project STR 'icos':
+        This will only return stilt stations that are ICOS stations.
+        
```

Comment:
please add example, to be consistent with the other help entries.

Reply:
Added example